python/cytokit_reader.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import csv
+import argparse
+import json
+
+
+def round_conv(s):
+    # TODO: Truncating after decimal point might be slightly too aggressive?
```

Comment:
I'll look into this with Arrow and compare file sizes.

Reply:
Perhaps for the x and y coordinates, we should have some precision beyond the decimal point, but I'm not convinced for the gene "levels" this is necessary.